hc/api/tests/test_sendalerts.py

```diff
@@ -35,4 +36,17 @@ def test_it_handles_grace_period(self):
         result = Command().handle_many()
         self.assertFalse(result)
 
-        
\ No newline at end of file
+    @patch("hc.api.management.commands.sendalerts.Command.handle_one")
+    def test_it_changes_next_nag_after_notification(self, mock):
+        check = Check(user=self.alice, status="down")
+        check.nag_time = timedelta(minutes=3)
+        check.grace = timedelta(minutes=1)
+        check.timeout = timedelta(minutes=1)
+        now = timezone.now()
+        check.next_nag = now
+        check.last_ping = now - timedelta(minutes=10)
+        check.save()
+        Command().handle_many()
+        checks = Check.objects.filter(status="down", nag_mode=True).first()
+        next_time = now + timedelta(minutes=3)
+        assert checks.next_nag >= next_time
```

Comment:
previous asserts follow the convention `self.assert`. Please use the same convention for this nag. Also isnt the time for next nag supposed to be exactly 3 minute later?